keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/TestController.kt

```diff
@@ -0,0 +1,21 @@
+package com.netflix.spinnaker.keel.rest
+
+import com.netflix.spinnaker.keel.yaml.APPLICATION_YAML_VALUE
+import com.netflix.spinnaker.kork.exceptions.SystemException
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/test"])
+class TestController {
+
+  @GetMapping(
+    path = ["/error"],
+    produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
+  )
+  fun throwAnException(): String {
```

Comment:
Would it be useful to support different types of exceptions/status codes? If so, you could add a parameter with the desired status code and throw an exception you know would cause that code to be returned.

Reply:
Yeah, I think it's potentially useful to support simulations of different kinds of errors, and I'd like to hold off on adding this functionality until we have tests that are consuming them (this meets my current testing needs for the exception alerts).